scripts/feedback_aggregator.py

```diff
@@ -468,6 +471,9 @@ def test_check_key(key):
     "archives/compressed/ff/01234567-89ab-cdef-0123-456789abcdef/2024Z4/08/073607.tar.gz",
     "archives/compressed/fZ/01234567-89ab-cdef-0123-456789abcdef/202404/08/073607.tar.gz",
     "archives/compressed/ff/01234567-89ab-cdZf-0123-456789abcdef/202404/08/073607.tar.gz",
+    "archives/compressed/00/00000000-0000-0000-0000-000000000001/20240/08/073607.tar.gz",
+    "archives/compressed/00/00000000-0000-0000-0000-000000000001/202404/8/073607.tar.gz",
+    "archives/compressed/00/00000000-0000-0000-0000-000000000001/202404/08/73607.tar.gz",
```

Comment:
i'm missing what's invalid about this last key?

also, not related to this PR, but why is the filename format "YYYYMM/DD" instead of "YYYY/MM/DD"?

Reply:
oh, it's invalid because the final time segment is 5, not 6, chars.

Reply:
last key: the time has not 6 digits
unrelated question: well it's how CCX service was configured at beginning because the original plan 4 years ago was that there will be daily batch processing and that number of objects in bucket is too big to store/compute diff between new objects and already processed objects. So IIRC it was preliminary optimization (and not a good one IMHO).